drivers/network/afd/afd/write.c

```diff
@@ -10,6 +10,8 @@
 
 #include "afd.h"
 
+#define PAD_BUFFER 2048;
```

Comment:
```suggestion
#define PAD_BUFFER 2048
```

Reply:
saw that right after I comitted and fixed